src/App.css

```diff
@@ -1,27 +1,321 @@
+}
+
+.book-title {
+  font-family: "Roboto Slab", serif;
+  font-weight: bold;
+  font-size: 1.375rem;
+  letter-spacing: -0.2px;
+  color: #121212;
+  margin: 0;
+}
+
+.book-author {
+  font-family: "Roboto Slab", serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+  margin: 0;
+}
+
+.book-content {
+  display: flex;
+  justify-content: space-between;
+}
+
+.button-outline {
+  font-family: "Roboto Slab", serif;
+  background-color: transparent;
+  border: none;
+  color: #4386bf;
+  font-size: 0.875rem;
+  font-weight: 300;
+  align-items: center;
+  cursor: pointer;
+}
+
+.disabled {
+  font-family: "Roboto Slab", serif;
+  background-color: transparent;
+  border: none;
+  color: #687580;
+  cursor: pointer;
+}
+
+.action-buttons {
+  display: flex;
+  align-items: center;
+  margin-top: 1.313rem;
+}
+
+.vertical-divider {
+  width: 0.125rem;
+  height: 1.5rem;
+  background-color: #e8e8e8;
+  margin: 0 1rem;
+}
+
+.progress-container {
+  display: flex;
+  align-items: center;
+}
+
+.progress-divider {
+  width: 0.125rem;
+  height: 4.375rem;
+  margin: 0 3.688rem 0 4.938rem;
+  background-color: #e8e8e8;
+}
+
+.percent-complete {
+  font-family: "Montserrat", sans-serif;
+  font-size: 2rem;
+  color: #121212;
+}
+
+.completed {
+  font-family: "Montserrat", sans-serif;
+  font-size: 00.875rem;
+  opacity: 0.5;
+  color: #121212;
+}
+
+.current-chapter-container {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  width: 14rem;
+}
+
+.current-chapter-label {
+  font-family: "Roboto Slab", serif;
+  font-weight: 300;
+  font-size: 0.813rem;
+  color: #121212;
+  opacity: 0.5;
+}
+
+.current-chapter {
+  font-family: "Roboto Slab", serif;
+  font-weight: 300;
+  margin-top: 0.438rem;
+  font-size: 1rem;
+  color: #121212;
+  letter-spacing: -0.4px;
+}
+
+.primary-button {
+  cursor: pointer;
+  border: none;
+  background-color: #0290ff;
+  border-radius: 3px;
+  padding: 8px 24px;
+  font-family: "Roboto Slab", serif;
+  font-weight: 300;
+  letter-spacing: 0.5px;
+  color: white;
+}
+
+.circular-progress-container {
+  width: 5.625rem;
+  height: 5.625rem;
+  padding: 0.63rem 0.625rem 0.75rem 0.75rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.circular-progress {
+  position: absolute;
+  border: solid 5px #307bbe;
+  border-left-color: #e8e8e8;
+  border-radius: 50%;
+  width: 4.25rem;
+  height: 4.25rem;
+  border-style: solid;
+  border-width: 5px;
+  transform: rotate(45deg);
+}
+
+@keyframes growProgressBar {
+  0%,
+  33% { --pgPercentage: 0; }
+  100% { --pgPercentage: var(--value); }
+}
+
+/* stylelint-disable */
+@property --pgPercentage {
+  syntax: '<number>';
+  inherits: false;
+  initial-value: 0;
+}
+
+/* stylelint-enable */
+
+div[role="progressbar"] {
+  --size: 4.24rem;
+  --fg: #307bbe;
+  --bg: #e8e8e8;
+  --pgPercentage: var(--value);
+
+  animation: growProgressBar 3s 1 forwards;
+  width: var(--size);
+  height: var(--size);
+  border-radius: 50%;
```

Comment:
* As stated in the project requirement `Your final design should match Zeplin's design.` currently, your design doesn't match the given design below are screenshots of highlighted areas in the design you should work on ( which are colours, margins, font sizes, and missing contents)
![image](https://user-images.githubusercontent.com/52497246/186859600-0f945145-aa14-4f00-b3fa-39a961d9c11e.png)

## YOURS 
![image](https://user-images.githubusercontent.com/52497246/186859646-e3bf6f62-fa53-43b9-a6c4-f98267f8572b.png)